Lib/threading.py

```diff
@@ -647,7 +647,8 @@ def wait(self, timeout=None):
         (or fractions thereof).
 
         This method returns the internal flag on exit, so it will always return
-        True except if a timeout is given and the operation times out.
+        True except if a timeout is given and the operation times out, when it
```

Comment:
```suggestion
        ``True`` except if a timeout is given and the operation times out, when it
```

Reply:
The double backticks on ``True`` make the line more than 80 characters, so I'll make the adjustment in a different commit.